assignments/closure.js

```diff
@@ -1,21 +1,43 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function sayHello(name) {
+  const text = (`Hello ${name}`);
+  const say = function() { console.log(text); }
+  say();
+}
+sayHello('Andy');
 
 /* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
 
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
+  let count = 0;
+  return(function() { return( ++count); })
```

Comment:
great job making use of javascript closure.
again you could make use of the ES6 syntax to clean up this function

```
const counter = () => {
  let count = 0;
  return () => ++count; 
}
```